src/OfflineMap.h

```diff
@@ -486,15 +488,17 @@ class OfflineMap {
 	///	<summary>
 	///		Set the mask vector associated with the source grid.
 	///	</summary>
-	void SetSourceMask(const DataArray1D<int> & iSourceMask) {
-		m_iSourceMask = iSourceMask;
+	void SetSourceMask(const std::vector<int> & iSourceMask) {
+		m_iSourceMask.Allocate(iSourceMask.size());
+		memcpy(&(m_iSourceMask[0]), &(iSourceMask[0]), iSourceMask.size() * sizeof(double));
 	}
 
 	///	<summary>
 	///		Set the mask vector associated with the target grid.
 	///	</summary>
-	void SetTargetMask(const DataArray1D<int> & iTargetMask) {
-		m_iTargetMask = iTargetMask;
+	void SetTargetMask(const std::vector<int> & iTargetMask) {
+		m_iTargetMask.Allocate(iTargetMask.size());
+		memcpy(&(m_iTargetMask[0]), &(iTargetMask[0]), iTargetMask.size() * sizeof(double));
```

Comment:
This is wrong! mask is an `int` vector. `sizeof(double)` should be `sizeof(int)`. Same above for source mask also

Reply:
Oof good catch